tests/host_tools/network.py

```diff
@@ -141,27 +201,6 @@ def _exec(self, cmd, timeout=100, check=False):
 
             raise
 
-    # pylint:disable=invalid-name
```

Comment:
I know this is not used in the integ tests, but it doesn't work anymore with this approach?

Reply:
Ah, I think this is mostly an artifact of the paramiko approach, where this wasn't supported anymore. Should still work here? But admittedly didnt look at it, since as you say its dead code :sweat_smile: